templates/start_services.j2

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+# {{ ansible_managed }}
+
+{% for item in certbot_create_standalone_stop_services %}
+echo "starting service {{ item }}"
+{% if ansible_service_mgr == 'systemd' %}
+systemctl start {{ item }}
```

Comment:
the start subcommand can take all designated service names as arguments at once, reducing the downtime due to fewer invoked processes. the same applies to the `stop_service.j2`.